src/libs/ReactNativeLauncher.js

```diff
@@ -309,7 +329,10 @@ class ReactNativeLauncher extends Launcher {
         throw initKonnectorError
       }
       await this.pilot.call('setContentScriptType', 'pilot')
-      await this.worker.call('setContentScriptType', 'worker')
+      const workerOptions = {
+        shouldRecord: flag('clisk.record')
```

Comment:
`shouldRecord` is defined but never used, is this handled inside of clisk library?

Reply:
Actually, this parameter is passed to the konnector itself which will handle the recording. 
This recording is automatically handled by the cozy-clisk lib, imported by all clisk konnectors